GoogleForCreators/web-stories-wp
# Color Picker accessibility improvements

## Context


## Summary
- Adds label to the swatches
- Adds keyboard navigation


## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions
- Open the color picker
- Tab through all the fields and verify the focus outline is displayed correctly
- Verify the swatches have aria-label
- Verify arrow keys can be used to navigate in a color group (e.g. within the Default colors), and tab can be used to move between the color groups





This PR can be tested by following these steps:

1.
## Checklist




---



Fixes #8657
